src/MoveRequestDialog.js

```diff
@@ -154,7 +154,7 @@ class MoveRequestDialog extends React.Component {
     query = `(${query}) and (status="Open")`;
     requests.reset();
 
-    return requests.GET({ params: { query } });
+    return requests.GET({ params: { query, limit: 1000 } });
```

Comment:
Is it seriously the case that if you ask for N records by ID, e.g. `itemId==foo or itemId==bar`, that you only get ten records, not N records, unless you specify a `limit` clause? 🤦 

Also, we need #672 in place first in order to be able to fetch more than ~40 rows, right?

Reply:
You know that without specifying a limit, we only get default = 10.

If the list of elements is split (40 for each chunk), there is a chance that we will also receive several hundred requests for each chunk.